sdk/energyservices/Azure.ResourceManager.EnergyServices/tests/Tests/EnergyServicesTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.EnergyServices.Models;
+using Azure.ResourceManager.EnergyServices.Tests.Helpers;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.EnergyServices.Tests.Tests
+{
+    [TestFixture]
+    public class EnergyServicesTests : EnergyServicesManagementTestBase
+    {
+        public EnergyServicesTests() : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                await CreateCommonClient();
+            }
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            //CleanupResourceGroups();
+        }
+
+        [TestCase]
```

Comment:
add `[Ignore("not record yet")]` to disable run test, then can pass the ci pipeline check